scheduler/src/cook/mesos/api.clj

```diff
@@ -176,6 +177,15 @@
    (s/optional-key :extract) s/Bool
    (s/optional-key :cache) s/Bool})
 
+(def MesosComputeCluster
+  "Schema for a compute cluster"
+  {:framework-id s/Str})
```

Comment:
Why is `:framework-id` the only thing in a `MesosComputeCluster`? Can't Cook connect to 2 different Mesos clusters using the same `framework-id`?

Reply:
MesosComputeCluster is a subkey under ComputeCluster, so the two are distinguished by having different compute-cluster-name, and (when we do multiple compute clusters) we'll have that available. In addition, we need the framework-id available in the task structure in the API response so that the cs cli has it available if/when it needs to talk to agents.

Reply:
When we do add support for multiple mesos clusters, we can either add additional information here (E.g., master) or read it from config. I'm not designing that part now. Which of those options we need to take depends mostly on what work cs cli needs to support multiple compute clusters.